### PR TITLE
Entities Settings.md

### PR DESCRIPTION
**Pull request by medhaanand**

### CODE REVIEW

File: docs/bot-builder/basic/entities.md [hunk 1]
```diff
@@ -540,19 +540,19 @@ So here we need to pass a unique value for every plan in the payload and store t
 
 1. Creating & storing the values in the entity
 
-   a. Create a Word & phrases entity 
+a. Create a Word & phrases entity 
 
 ![ES_6](assets/ES_6.png)
 
-   b. Click on Advanced options and enter the **Payload_keys** & **Context_keys** you want as per your use case, I have used **plan** & **planid** for purchase a plan use case
+b. Click on Advanced options and enter the **Payload_keys** & **Context_keys** you want as per your use case, I have used **plan** & **planid** for purchase a plan use case
```
Comment (shrey4500) [inline docs/bot-builder/basic/entities.md:547]: We can't use first person in documentation, right ? "I have used" should be replaced with "The below examples uses..."

File: docs/bot-builder/basic/entities.md [hunk 2]
```diff
@@ -540,19 +540,19 @@ So here we need to pass a unique value for every plan in the payload and store t
...
 
 ![ES_7](assets/ES_7.png)
 
-  c. Click on the Dictionary option present in the top and enter all Entity values you need for your buttons. In my case, as I have 4 plans so I created 4 dictionary values with proper variations as shown below
+c. Click on the Dictionary option present in the top and enter all Entity values you need for your buttons. In my case, as I have 4 plans so I created 4 dictionary values with proper variations as shown below
 
 ![ES_8](assets/ES_8.png)
 
-  d. Save the entity and click on Add to bot says to add the entity on that node.
+d. Save the entity and click on Add to bot says to add the entity on that node.
```
Comment (shrey4500) [inline docs/bot-builder/basic/entities.md:555]: "Add to Bot Says" in quotes.

File: docs/bot-builder/basic/entities.md (hunk 1 above)
Comment (medhaanand) [reply docs/bot-builder/basic/entities.md]: Made the changes accordingly

File: docs/bot-builder/basic/entities.md (hunk 2 above)
Comment (medhaanand) [reply docs/bot-builder/basic/entities.md]: Done

File: docs/bot-builder/basic/entities.md [hunk 3]
```diff
@@ -447,6 +448,148 @@ Below is the case where the detection stopped once the Negative Variations were
...
 
+### **Entity Settings**
+
+We have 4 main Advanced Settings options under entities.
+
+**1. Previous Context Tags**
+
+**What is Context from an IVA perspective?**
+
+Context is linguistically defined as the background in which the conversation is taking place. From the IVA perspective, it can involve retaining the memory of any piece of information that has been either derived from past user behavior, the action performed by the IVA, or has been explicitly given by the user.
```
Comment (tejas-haptik) [inline docs/bot-builder/basic/entities.md:459]: `behaviour`

**Pull request merged**